python/tests/mock_vim.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import sys
+import os
+import mock
+
+def mock_eval(expr):
+    '''Mock vim.eval function.
+
+    This will have hard-coded cases, which represent likely values returned by
+    the actual vim implementation
+
+    Args:
+        expr (str): The expression that vim would evaluate
+    '''
+    if expr.startswith('line('):
+        return "9"
+    elif expr.startswith('col('):
+        return "16"
+    elif expr.startswith('getline('):
+        return ["mybufferdata"]
+    elif expr.startswith('g:OmniSharp_timeout'):
+        return 123
```

Comment:
Could you add `elif expr.startswith('g:OmniSharp_translate_cygwin_wsl'): return 0` here? Then we'll have an initially passing test suite.